src/containers/HomePage/HomePage.jsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import Footer from "../../components/Footer/Footer";
+
+function HomePage() {
+  return (
+    <>
+      <Footer />
```

Comment:
Since the footer exists in all the pages (not just homepage) I think it should be rendered in the app.js file

Reply:
moved footer to App as requested